semantic_conventions.md

```diff
@@ -44,6 +44,7 @@ Every Span log has a specific timestamp (which must fall between the start and f
 | `error.kind` | string | The type or "kind" of an error (only for `event="error"` logs). E.g., `"Exception"`, `"OSError"` |
 | `error.object` | object | For languages that support such a thing (e.g., Java, Python), the actual Throwable/Exception/Error object instance itself. E.g., A `java.lang.UnsupportedOperationException` instance, a python `exceptions.NameError` instance |
 | `event` | string | A stable identifier for some notable moment in the lifetime of a Span. For instance, a mutex lock acquisition or release or the sorts of lifetime events in a browser page load described in the [Performance.timing](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming) specification. E.g., from Zipkin, `"cs"`, `"sr"`, `"ss"`, or `"cr"`. Or, more generally, `"initialized"` or `"timed out"`. For errors, `"error"` |
+| `level` | string | Levels used for identifying the severity of an event. Levels are organized from most specific to least: `"FATAL"`(most specific, little data), `"ERROR"`, `"WARN"`, `"INFO"`, `"DEBUG"` and `"TRACE"`(least specific, a lot of data) |
```

Comment:
@yurishkuro I can change on that. 

"Error", "warn", "info", "debug" are the most common level for me. Log component library, log4j2, gives more than these 4, like "trace", "all" etc. I agree you on this.